### ISSUE TITLE
I'd like to take over development for passport discord

### ISSUE THREAD

**Issue by AfaGaming** [nicholastay/passport-discord#54]
On this website: https://www.passportjs.org/packages/passport-discord/

You said the following:

"Notice: I'm no longer too active with the Discord API, and only tinker around occasionally. So, if there is anybody who would like to be more active in maintaining, I'm happy to link to your fork as the new solution to use or give project permissions on this repo."

I would like to take over and maintain passport discord.

**Comment by Voroniyx** [comment]
I think the best first step would be for them to do a fork and just get started

**Comment by laGameTV** [comment]
Yes, especially because the whole package will probably break soon when Discord introduces the new usernames.

**Comment by slekup** [comment]
I made [passport-discord-auth](https://www.npmjs.com/package/passport-discord-auth), which is really the same thing, just with updated variables and typescript. Also down to add some stuff and give repo permission to anyone who wants to as well.

This package can still work though, same as the oauth-everything one, but they haven't been updated in time.
